spec.md

```diff
@@ -8,121 +8,160 @@ title: Semantic Specification
-- `Inject` a `Span` instance into a "carrier" object for cross-process propagation. The type of the carrier is either determined through language reflection or an explicit [format](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
-- `Join` a Trace given a "carrier" object whose contents crossed a process boundary. `Join` examines the carrier and tries to extract identifying information needed by the new `Span` instance it's trying to create. `Join` also accepts an operation name for that `Span` (since it's typically not included within the carrier). Unless there's an error, `Join` returns a freshly-started `Span` which can be used in the host process like any other. (Note that some OpenTracing implementations consider the `Span`s on either side of an RPC to have the same identity, and others consider the caller to be the parent and the receiver to be the child) The type of the carrier is either determined through language reflection or an explicit [format](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
-
-Note that `Inject` and `Join` are not entirely symmetrical interfaces since `Span` injection is a fundamentally lossy process (most of the `Span`'s state is meant to be recorded out-of-band from the application's own communication path, whereas Inject-ed state is meant to be sent in-band along with the application data).
-
+- **Start a new `Span`**. The caller can specify zero or more [`SpanContext` references](#references) (e.g., `Follows` or `BelongsTo`), an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpan`)**
+- <span id="inject-extract"></span>**Inject a `SpanContext`** into a "carrier" object for cross-process propagation. The type of the carrier is either determined through reflection or an explicit [format identifier](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
+- **Extract a `SpanContext`** given a "carrier" object whose contents crossed a process boundary. Extract examines the carrier and tries to reconstruct the previously-Injected `SpanContext` instance. Unless there's an error, Extract returns a `SpanContext` which can be used in the host process like any other, most likely to start a new (descendant) Span. (Note that some OpenTracing implementations consider the `Span`s on either side of an RPC to have the same identity, and others consider the caller to be the ancestor and the receiver to be the descendant) The type of the carrier is either determined through language reflection or an explicit [format](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
```

Comment:
> Note that some OpenTracing implementations consider the `Span`s on either side of an RPC to have the same identity, and others consider the caller to be the ancestor and the receiver to be the descendant

For implementations that consider the receiver to be the descendant, is Extract supposed to return a new SpanContext with a new span ID, descending from the extracted parent context?